### DISCUSSION TITLE
"Merge" #853 from SE-2

### DISCUSSION THREAD

**Discussion by ByteAtATime** [BuidlGuidl/abi.ninja#116] [Ideas]
I recently authored https://github.com/scaffold-eth/scaffold-eth-2/pull/853, which aims to majorly improve the transaction result UI. I think I think it would be useful to merge that into abi.ninja. I use the word "merge" loosely here, because I understand abi.ninja isn't based on SE-2, but I think there's not many changes from SE-1 to SE-2.

**Comment by carletex** [comment]
> because I understand abi.ninja isn't based on SE-2

It is! Just an _older_ version of it (still using the NextJS pages router, old wagmi/viem versions, etc)

I think it's a good idea to improve the tx result UI. Go for it.

**Reply by carletex** [reply]
Oh and

> But I think there's not many changes from SE-1 to SE-2.

You should really check the SE-1 codebase and tell me :P

**Reply by ByteAtATime** [reply]
1. Ah, I was looking at only the abi.ninja directory, not the actual SE-1. I didn't realize you guys merged the initial tx result components. Also, it says "generated from scaffold-eth/scaffold-eth", not scaffold-eth-2.
2. I was moreso referencing the fact that the abi.ninja tx result, haha.

Anyway, I will get to work later.
